cron_job/cron_job_params

```diff
@@ -0,0 +1,10 @@
+# .---------------- minute (0 - 59)
+# | .------------- hour (0 - 23)
+# | | .---------- day of month (1 - 31)
+# | | | .------- month (1 - 12)
+# | | | | .---- day of week (0 - 6) (Sunday=0 or 7)
+# | | | | |
+# * * * * * command
+
+0 4 * * * docker run -d --rm stop_spot python3 StopSpot_Data_Pipeline/main.py --daily
```

Comment:
So to be clear this line would have the daily script run every 4 hours?

Reply:
This line would run the script at 4am everyday.

Reply:
Actually it would run this command at 4 am everyday which would run the pipeline by running the application with the daily flag to be more specific.

Reply:
Also I forgot to mention that when running the pipeline it would process the next day's data hence the "daily" flag.